### ISSUE TITLE
Maybe we need a wire protocol for controlling capture

### ISSUE THREAD

**Issue by bradgrantham-lunarg** [LunarG/gfxreconstruct#870]
Something to control capture, like "start trimming" and "stop trimming" would be great.. 

Because there would be a little delay connecting to the app maybe also an option to capture to cause GFXR to wait in CreateInstance and a wire signal for "control app connected" (GFXR can continue) would be helpful.

Maybe the control app would be a Python script with Tcl/Tk.  I don't want to think much more about it for fear of overengineering it before we even commit to such a thing.

**Comment by andrew-lunarg** [comment]
Sounds great. Maybe we can stretch to some prettier buttons than Tk's :pray: 
<https://github.com/hoffstadt/DearPyGui>
<https://kivy.org/>

**Comment by bradgrantham-lunarg** [comment]
Could do.  Mostly I was thinking the Python/Tk app probably would run on any installation of Python so we don't add any new dependencies.

**Comment by andrew-lunarg** [comment]
True. TkInter actually got a bit prettier since I tried it ... some time
before the 2007 revamp I guess :-).

**Comment by bradgrantham-lunarg** [comment]
I think TkInter themes help them look a little better.

![image](https://user-images.githubusercontent.com/50641407/197635945-c8fe5ea5-103a-49fa-a41d-1012e4786c4b.png)

**Comment by panos-lunarg** [comment]
What's the basic idea for this? Are we going to bese it on adb (android specific) or something more general like a tcp socket?
